### ISSUE TITLE
Unrecognized Header

### ISSUE THREAD

**Issue by lfuertes** [loopj/android-async-http#1048]
Hi, I imported the jar and I copied the example of the documentation, but Android Studio not recognize me the Header. Can you tell me why? :(

        AsyncHttpClient client = new AsyncHttpClient();
        client.get("https://www.google.com", new AsyncHttpResponseHandler() {

            @Override
            public void onStart() {
                // called before request is started
            }

            @Override
            public void onSuccess(int statusCode, Header[] headers, byte[] response) {
                // called when response HTTP status is "200 OK"
            }

            @Override
            public void onFailure(int statusCode, Header[] headers, byte[] errorResponse, Throwable e) {
                // called when response HTTP status is "4XX" (eg. 401, 403, 404)
            }

            @Override
            public void onRetry(int retryNo) {
                // called when request is retried
            }
        });

**Comment by rpelorosso** [comment]
The same thing is happening to me. Have you had any luck resolving this?

**Comment by lfuertes** [comment]
Yes, I have had to import on the Gradle as placed on the documentation, not like android studio modules settings suggests.

On Gradle(Module: app): compile 'com.loopj.android:android-async-http:1.4.9'

And for recognize Header import in Activity this: import cz.msebera.android.httpclient.Header;

Regards

**Comment by luchfilip** [comment]
Thank you @lfuertes , you saved me a headache!
